packages/http-server/test/integration/http-server.integration.ts

```diff
@@ -134,6 +137,13 @@ describe('HttpServer (integration)', () => {
     expect(anotherServer.start()).to.be.rejectedWith(/EADDRINUSE/);
   });
 
+  it('supports HTTPS protocol with key and certificate files', async () => {
+    const httpsServer: HttpServer = givenHttpsServer();
+    await httpsServer.start();
+    const response = await httpsGetAsync(httpsServer.host!, httpsServer.port);
```

Comment:
I would prefer `httpsGetAsync` to accept URL and then internally use `url.parse` to decompose the provided URL into `https.request` options. Not a big deal though.

One thing to verify - how does `httpsServer.host` work with IPv6 addresses. Please add a test making an HTTPS request to `::1` (IPv6 loopback).

Reply:
> One thing to verify - how does `httpsServer.host` work with IPv6 addresses. Please add a test making an HTTPS request to `::1` (IPv6 loopback).

PTAL, I think you forgot to add this test.